nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -158,6 +158,9 @@ bool BtActionServer<ActionT>::on_configure()
   int default_server_timeout;
   node->get_parameter("default_server_timeout", default_server_timeout);
   default_server_timeout_ = std::chrono::milliseconds(default_server_timeout);
+  int bt_wait_for_service_timeout;
+  node->get_parameter("bt_wait_for_service_timeout", bt_wait_for_service_timeout);
```

Comment:
Great! I think this needs to be added as a parameter in the parameter  guide for the BT Navigator server and that should be good to go! Here's the file: https://github.com/ros-planning/navigation.ros.org/blob/master/configuration/packages/configuring-bt-navigator.rst